CAIPs/caip-69.md

```diff
@@ -0,0 +1,72 @@
+---
+caip: 69
+title: EOA extension to CAIP-10
+author: Griff Green (@griffgreen), Krati Jain (@jainkrati), Greg-the-Greek (Please edit Greg!)
+status: Draft
+type: Standard
+created: 2024-03-07
+---
+
+
+## Simple Summary
+
+CAIP-10 defines a way to identify an address in our multichain world using
+CAIP-2 blockchain id. CAIP-69 extends it to include a special case for EOAs
+using the chain ID of 0.
+
+
+## Abstract
+
+This proposal aims to facilitate specifying EOAs as multichain accounts
+extending CAIP-10 chain id specification. This is useful for both
+decentralized applications to communicate that EOAs can be used for multiple 
+chains using a chainID of 0 to specify that the account is an EOA. Currently, 
+addresses are usually assigned to only one chain using CAIP-10 and an EOA
+would be required to be assigned multiple times for each chain. This proposal 
+aims to extend the CAIP-10 standard so EOAs can be designated as an EOA once,
+and be treated as an EOA on every applicable chain.
+
+
+## Motivation
+
+The motivation for proposal stems from the redundancy of assigning an EOA
+as living on every chain that exists. As new chains continue to be launched
+it becomes more and more obvious that assigning an EOA its own specific
+designation will greatly simplify their use in cross chain applications.
+
+
+## Specification
+
+The account id specification will follow CAIP-10, but for EOAs use the 
+prefix `eip155:10:` before teh EOA account address
```

Comment:
Potential type, do you mean `0` instead of `10`.

I would suggest also adding the case for `1` to be designated as a smart account?

Reply:
+1

Reply:
Greg's idea of representing contracts or non-EOA accounts also sounds good. as per CAIP-10, eip155:1:<address> represents ethereum mainnet address. so the case for 1 is already taken. we would need alternative for the case of contract.

Reply:
Actually we're fine, because CAIP-10 only contains one colon, where this CAIP contains two. So you would just need to pre-check the string for number colons to perform the match :)

Reply:
> Potential type, do you mean `0` instead of `10`.
YEP! Fixing!


> I would suggest also adding the case for `1` to be designated as a contract, or non-eoa account? If so lets also add that to the examples

Hmmmm would there be a challenge for backwards compatibility? I think its cleaner to reserve the chainID of 0 for EOAs than to add another param to the standard... however, it might make sense if this is a pattern that we can use for other ecosystems like cosmos or polkadot.

Reply:
yes, in Test Cases section in https://chainagnostic.org/CAIPs/caip-10, all addresses have two colons. Would appreciate if you could clarify if there are gaps in our understanding @GregTheGreek